lib/improver/tests/utilities/test_load.py

```diff
@@ -150,6 +155,48 @@ def test_ordering(self):
         self.assertArrayAlmostEqual(result.coord_dims("latitude")[0], 2)
         self.assertArrayAlmostEqual(result.coord_dims("longitude")[0], 3)
 
+    def test_ordering_for_percentile_over_coordinate(self):
+        """Test that the cube has been reordered, if it is originally in an
+        undesirable order and the cube contains a "percentile_over"
+        coordinate."""
+        directory = mkdtemp()
+        filepath = os.path.join(directory, "temp.nc")
+        cube = set_up_percentile_temperature_cube()
+        percentile_points = np.array([10, 50, 90])
+        cube.transpose([3, 2, 1, 0])
+        iris.save(cube, filepath)
+        result = load_cube(filepath)
+        self.assertIsInstance(result, iris.cube.Cube)
+        self.assertArrayAlmostEqual(
+            result.coord("percentile_over_realization").points,
+            percentile_points)
+        self.assertArrayAlmostEqual(
+            result.coord("time").points, self.time_points)
+        self.assertArrayAlmostEqual(
+            result.coord("latitude").points, self.latitude_points)
+        self.assertArrayAlmostEqual(
+            result.coord("longitude").points, self.longitude_points)
+
+    def test_ordering_for_threshold_coordinate(self):
+        """Test that the cube has been reordered, if it is originally in an
+        undesirable order and the cube contains a "threshold" coordinate."""
+        directory = mkdtemp()
+        filepath = os.path.join(directory, "temp.nc")
```

Comment:
Duplicate code. Remove these two lines and replace use of filepath with self.filepath (from setUp()).

Reply:
Removed.